cmd/tailscale/main.go

```diff
@@ -704,8 +759,8 @@ func (a *App) isChromeOS() bool {
 func (s *BackendState) updateExitNodes() {
 	s.ExitStatus = ExitNone
 	var exitID tailcfg.StableNodeID
```

Comment:
This looks like it's read/written from different goroutines, so probably better to make it an `atomic.Pointer` or guard it with a mutex.